src/elfpy/pricing_models/base.py

```diff
@@ -375,17 +373,17 @@ def get_max_long(
         float
             The maximum amount of base that can be used to purchase bonds.
```

Comment:
we're returning a tuple here right? of `(maybe_max_long, d_bonds)` why is this necessary? the agent ignores the latter, right?